android/build.gradle.kts

```diff
@@ -21,19 +21,23 @@ android {
     compileSdk = libs.versions.compileSdk.get().toInt()
 
     defaultConfig {
-        applicationId = "co.touchlab.droidcon.london"
+        applicationId = "co.touchlab.droidconauthtest"
         minSdk = libs.versions.minSdk.get().toInt()
         targetSdk = libs.versions.targetSdk.get().toInt()
         versionCode = 20201
         versionName = "2.2.0"
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+
+        val clientId = properties.getProperty("clientId", "")
+        buildConfigField("String", "CLIENT_ID", clientId)
```

Comment:
Can you update the readme with instructions on how to set your clientID for both android and iOS?

Reply:
Updated the readme